controller/src/main/java/io/pravega/controller/store/stream/tables/serializers/HistoryRecordSerializer.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Please do not reformat the license header.

Reply:
Done